.travis.yml

```diff
@@ -8,6 +8,9 @@ install:
 notifications:
   slack:
     rooms:
-      - undeadbaby:$SLACK_KEY#random
+    - undeadbaby:WDGNd6FP2Uu61jNcbMaHLSIv#random
```

Comment:
Is the key substitution not working correctly?

keys in plain text stored in a repo is very bad form, even more so when the repository is public like this one. 
In this case it's not super serious, cause it just grants access to our slack which is hardly sensitive, but it'd be good to have key substitution working correctly in the case we do need to use something sensitive in the future.